### DISCUSSION TITLE
Missing option to configure "Password login alert" email

### DISCUSSION THREAD

**Discussion by ColinSchmale** [pocketbase/pocketbase#5185] [Q&A] [answered]
I can't find an option to configure the "Password login alert" email. I've read in https://github.com/pocketbase/pocketbase/discussions/5104 that you plan to change it to "Login from new device" emails. It would still be nice if we could change the template for those emails.

**Comment by ganigeorgiev** [comment, marked as answer]
It is not configurable and there are no plans to enable it for v0.22.x releases.

As mentioned in the linked discussion, the login alert email is just a temp workaround and with the next v0.23.0 release (there are no ETAs yet) it will be replaced with "Login from new device" email template (it is configurable).
